iis-schemas/src/main/avro/eu/dnetlib/iis/importer/ProjectToOrganization.avdl

```diff
@@ -0,0 +1,10 @@
+@namespace("eu.dnetlib.iis.importer.schemas")
+protocol IIS{
+
+    record ProjectToOrganization {
+//      Project identifier
```

Comment:
Fixed, although this is one convention against other - most of the schemas are defined without leading indentation.